articles/sso/current/setup.md

```diff
@@ -1,56 +1,62 @@
 
-For Social Identity Providers ensure the Connection is not using [developer keys](/connections/social/devkeys).
+For Social Identity Providers, make sure the Connection is not using [developer keys](/connections/devkeys).
 
-## 2. Enable SSO for the Application
+## 2. Configure SSO
 
-Navigate to the Applications section of the [Dashboard](${manage_url}/#/applications). Click on **Settings** (represented by the gear icon) for the Application to enable SSO on.
+Auth0 maintains an SSO session for any user authenticating via that Application. Auth0 maintains two pieces of information:
 
-![](/media/articles/sso/single-sign-on/clients-dashboard.png)
+| Setting | Description |
+| - | - |
+| Inactivity timeout | The maximum length of time that can elapse without user activity before the user is asked to log in again. **This setting cannot exceed 3 days!** |
+| Require log in after | The length of time that elapses before Auth0 forces the user to log in again (regardless of activity) |
 
-Near the bottom of the **Settings** page, toggle **Use Auth0 instead of the IdP to do Single Sign On**.
+To configure the **SSO Cookie Timeout** setting, navigate to [Dashboard > Tenant Settings > Advanced](${manage_url}/#/tenant/advanced).
 
-![](/media/articles/sso/single-sign-on/sso-flag.png)
+![](/media/articles/sso/sso-session-mgmt-2.png)
 
-::: note
-You can also set the Application's SSO flag using the [Auth0 Management API](/api/management/v2#!/Clients/patch_clients_by_id).
-:::
+Please note that any time a user performs a new standard login resets the SSO session.
 
-## 3. Configure SSO Session Length
+### Addendum: SSO Configuration for Legacy Tenants
 
-When the SSO flag is set for an Application, Auth0 maintains an SSO session for any user authenticating via that Application. The **SSO Cookie Timeout** setting determines how long an SSO session is valid. By default, an SSO session expires in 10080 minutes (or 7 days).
+In addition to the settings available under tenant settings, legacy tenants may see slightly different options available for SSO under [Dashboard > Tenant Settings > Advanced](${manage_url}/#/tenant/advanced).
 
-To configure the **SSO Cookie Timeout** setting, navigate to [Dashboard > Tenant Settings > Advanced](${manage_url}/#/tenant/advanced).
+![](/media/articles/sso/sso-session-mgmt-1.png)
 
-![](/media/articles/sso/single-sign-on/accountsettings-ssotimeout.png)
+While all new Auth0 tenants come with seamless SSO enabled, legacy tenants may choose whether to enable this feature.
 
-SSO session cookies expire after **3 days** of inactivity. For example, if no application (in the same browser) on a user's machine performs a login using the SSO session then the cookie expires after 3 days, even though a server side session might persist. Performing a new standard login would reset the SSO session.
+If you opt to **Enable Seamless SSO**, you have an additional setting available to you under Application Settings.
```

Comment:
This is backwards.  If you do not Enable Seamless SSO, you have the setting.  If you DO enable Seamless SSO then the setting is removed.